docs/sources/alerting/unified-alerting/alerting-rules/create-cortex-loki-managed-recording-rule.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Create Cortex or Loki managed recording rule"
+description = "Create Cortex or Loki managed recording rule"
+keywords = ["grafana", "alerting", "guide", "rules", "recording rules", "create"]
+weight = 400
++++
+
+# Create a Cortex or Loki managed recording rule
```

Comment:
Do we actually call Metrics and Logging by the names Cortex and Loki in Cloud docs as well as OSS Docs?

Reply:
> Do we actually call Metrics and Logging by the names Cortex and Loki in Cloud docs as well as OSS Docs?

@richardqlam, can you shed more light on this. Will help us working on OSS docs as well. Thank you.

Reply:
Talked to @Eve832 about this but in general, we like to talk about Cloud metrics and Cloud logs in a broad sense. It's okay to use the specific OSS terminology where it's relevant though, because sometimes it's necessary.  

For example, let's first consider metrics. It's not totally relevant to users that we use Cortex as our metrics backend in Grafana Cloud. What matters, though, is that "Grafana Cloud metrics are compatible with Prometheus and Graphite metrics". 

But when you look at Grafana Cloud logs, the story is a bit different. It _requires_ you to understand the kind of logging system we're using (Loki), because the logs are structured completely different than a system like Elastic and fit a totally different use case.